book/src/proptest-derive/modifiers.md

```diff
@@ -34,20 +34,31 @@ of [`prop_filter`].
 Example:
 
 ```rust
+# extern crate proptest_derive;
+# extern crate proptest;
+# use proptest_derive::Arbitrary;
+# use proptest::prelude::*;
```

Comment:
what do all these pound signs do?

Reply:
I take it from reading this later `// doctests don't run under cfg(test), so we need to do this.` that this hides this code in the docs but it's useful for compilation of the doc test?

Reply:
Yeah. This is a weird hack,  but you got the main idea.

Pound signs hide these lines from the rendered docs, so we just show the bit that's useful for people to see.

And because doctests aren't compiled under cfg test, the test cases don't get defined and can't be called from main unless we also define them with cfg not test.